search/elasticsearch/util/src/java/org/sakaiproject/search/elasticsearch/SiteElasticSearchIndexBuilder.java

```diff
@@ -113,14 +113,15 @@ protected void beforeElasticSearchConfigInitialization() {
             this.suggestionResultFieldNames = new String[] {
                     SearchService.FIELD_TYPE,
                     SearchService.FIELD_REFERENCE,
-                    SearchService.FIELD_SITEID,
```

Comment:
Why replacing it instead of adding one? Just curious.

Reply:
Because I'm an idiot